Core/MobileCore.cs

```diff
@@ -0,0 +1,181 @@
+
+            if (injector != null && options.ConfigFileName != null)
+            {
+                var filename = $"{injector.DefaultResources}.{options.ConfigFileName}";
+                try
+                {
+                    using (var stream = injector.ExecutingAssembly.GetManifestResourceStream(filename))
+                    {
+                        servicesConfig = MobileCoreJsonParser.Parse(stream);
+                    }
+                }
+                catch (System.Exception e)
+                {
+                    throw new InitializationException($"{filename} could not be loaded", e);
+                }
+            }
+            else
+            {
+                if (options.ConfigJson != null)
+                {
+                    try
+                    {
+                        MobileCoreJsonParser.Parse(options.ConfigJson);
+                    }
+                    catch (System.Exception e)
+                    {
+                        throw new InitializationException("invalid JSON configuration file", e);
+                    }
+                }
+                else
+                    throw new InitializationException("Must provide either filename or JSON configuration in Init() options");
+            }
+
+            if (options.HttpServiceModule == null)
+            {
+                HttpClient httpClient = new HttpClient();
+                httpClient.Timeout = TimeSpan.FromSeconds(DEFAULT_TIMEOUT);
+                var httpServiceModule = new SystemNetHttpServiceModule(httpClient);
+                var configuration = GetServiceConfiguration(httpServiceModule.Type);
+                if (configuration == null)
+                {
+                    configuration = ServiceConfiguration.Builder.Build();
+                }
+                httpServiceModule.Configure(this, configuration);
+                HttpLayer = httpServiceModule;
+            }
+            else HttpLayer = options.HttpServiceModule;
+        }
+
+        public static MobileCore Init() => Init(null, new Options());
+
+        public static MobileCore Init(IPlatformInjector injector) => Init(injector, new Options());
+
+        public static MobileCore Init(IPlatformInjector injector, Options options)
+        {
+            Contract.Requires(options != null);
+            instance = new MobileCore(injector, options);
+            return instance;
+        }
+
+        /// <summary>
+        /// Called when mobile core instance needs to be destroyed.
+        /// </summary>
+        public void Destroy()
+        {
+            foreach (var serviceModule in services.Values)
+            {
+                serviceModule.Destroy();
+            }
+            instance = null;
+        }
+
+        /// <summary>
+        /// Returns instance of a service module.
+        /// </summary>
+        /// <typeparam name="T">service module type</typeparam>
+        /// <param name="serviceClass">service module class type</param>
```

Comment:
This param description can probably be removed.

Reply:
yes, it's a reminder of older method implementation